<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sortWidget</class>
 <widget class="QWidget" name="sortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="returnButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Back To Menu</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="busRadioButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Bus</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="truckRadioButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Truck</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="carRadioButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Car</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="sortButton">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Sort</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>HarmonyOS Sans SC</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>                Unit:yuan</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
